core/workspace_svg.js

```diff
@@ -421,6 +421,10 @@ Blockly.WorkspaceSvg.prototype.inverseScreenCTMDirty_ = true;
  * @return {!Blockly.blockRendering.Renderer} The renderer attached to this workspace.
  */
 Blockly.WorkspaceSvg.prototype.getRenderer = function() {
+  if (!this.renderer_) {
+    this.renderer_ =
+        Blockly.blockRendering.init(this.options.renderer || 'geras');
```

Comment:
I don't mind the lazy load, but what's the reasoning?

Reply:
It looks like there's some funny load order going on with Workspace and WorkspaceSvg when setting the cursor.
If we reserve the setCursor / setMarker methods for dynamic setting of the cursor / marker (ie: after the workspace has been initialized), we can then set ``this.cursor_ = new Blockly.Cursor()`` in the constructor of Workspace instead of calling ``setCursor``. 

This moves the initialization of the drawer over to Workspace_svg's createDom method where the cursor is being injected into the DOM.

Reply:
I explained a bit in the PR description, but essentially the issue is that the renderer is normally initialized after the setCursor/setMarker methods are called, but I need to use it inside of the methods.
Your suggestion of reserving setCursor/setMarker for dynamically changing them works. It'll also make it more clear exactly when the renderer is initialized.

Reply:
Yeah I recognize the issue you're running into here, and I think the problem lies in the fact we were calling overridden methods from the constructor (making it hard to trace the initialization order) but more importantly assuming the renderer has been initialized.

Reply:
Applied fix